app/helpers/review_count_helper.rb

```diff
@@ -0,0 +1,13 @@
+module ReviewCountHelper
+  def cards_to_review(show_text: false, text: "Осталось вспомнить ")
+    to_remind = Rails.cache.fetch("cards_to_review")
+    if show_text
+      content_tag :p do
+        concat(text)
+        concat(content_tag(:span, to_remind, class: 'badge'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.